### ISSUE TITLE
Shelling-in with `emp run` requires a return before accessing the shell

### ISSUE THREAD

**Issue by benjamin-guillet** [remind101/empire#609]
According to @ejholmes it's probably some output stuck in a buffer (that needs to be flushed) or a docker API issue. Needs investigation.

**Comment by ejholmes** [comment]
@benguillet have you seen this happen recently? I can't recall running into this for quite a while (maybe we inadvertently fixed it by upgrading some dep?).

**Comment by benguillet** [comment]
@ejholmes Yeah, it's pretty rare but i got a few of them recently, on postoffice mostly

**Comment by ejholmes** [comment]
I got annoyed enough with this today to investigate a bit. Apparently, this is mostly a Docker issue. I can replicate it completely with:

```bash
docker attach $(docker run -d -it ubuntu bash)
```

https://asciinema.org/a/5n9gj0p6fjm9b8kxnhog4fzl1

The linked PR at the bottom suggests that resizing the TTY might fix it, so we could do that.
